test/common/index.js

```diff
@@ -568,9 +568,23 @@ exports.canCreateSymLink = function() {
   return true;
 };
 
+exports.getCallSite = function getCallSite(top) {
```

Comment:
Would it be better if we validated that if `top` is actually a function?

Reply:
It doesn't really matter, in my opinion. The only place it is used is in `Error.captureStackTrace()` which does that checking for us. I.e. if it's not a function, the parameter is ignored and the top frame in the stack will be where `Error.captureStackTrace()` was called.

```js
> const common = require('./test/common');
undefined
> let x = common.getCallSite()
undefined
> x
'/Users/lanceball/src/node/test/common/index.js:576'
> x = common.getCallSite('abc');
'/Users/lanceball/src/node/test/common/index.js:576'
> x = common.getCallSite(123)
'/Users/lanceball/src/node/test/common/index.js:576'
>
```